src/main/java/serverutils/mixin/Mixins.java

```diff
@@ -23,7 +22,10 @@ public enum Mixins {
     REPLACE_TAB_NAMES(new Builder("Replace tab menu names").addTargetedMod(VANILLA).setSide(Side.CLIENT)
             .setPhase(Phase.EARLY).addMixinClasses("forge.MixinGuiIngameForge")),
     VANILLA_TP_BACK_COMPAT(new Builder("/back compat for the vanilla /tp").addTargetedMod(VANILLA).setSide(Side.BOTH)
-            .setPhase(Phase.EARLY).setApplyIf(() -> commands.back).addMixinClasses("minecraft.MixinCommandTeleport")),;
+            .setPhase(Phase.EARLY).setApplyIf(() -> commands.back).addMixinClasses("minecraft.MixinCommandTeleport")),
+    PLAYERS_SLEEPING_PERCENTAGE(new Builder("Player Sleeping Percentage").addTargetedMod(VANILLA).setSide(Side.BOTH)
```

Comment:
maybe Open to lan

Reply:
Does it need to apply on both in order to work with open to lan? I don't particularly care about supporting that feature, but that's the only use case I can think of. Otherwise it could be changed to server only.

Reply:
Turning off the chat notifications when there's only one player present could also work to prevent unnecessary messages in SP, but I don't really care much for lan either so changing it to server only is also fine by me.